### PR TITLE
Update Winbox to 3.37

### PR DESCRIPTION
**Pull request by juanpedrojose**
Update Winbox x64 executable to 3.37

### CODE REVIEW

File: CHANGELOG_WINBOX.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Winbox changelog
 
+## 3.37
+
+- do not update interface window layout all the time;
+- do not send value for empty multi-option field;
+- fixed VPLS "cisco-id" field with "&" prefix (introduced v3.36);
+- fixed ability to set options that can be hidden;
+- fixed floating point field's negative value in `-0.***` format;
```
Comment (nrlquaker) [inline CHANGELOG_WINBOX.md:9]: Please change `;` to `.`.

Comment (nrlquaker) [reply CHANGELOG_WINBOX.md:9]: In the last line only.

Comment (juanpedrojose) [reply CHANGELOG_WINBOX.md]: Fixed

**Comment by nrlquaker** [comment]
@juanpedrojose thanks for contribution.

**Pull request merged**